src/movies.js

```diff
@@ -1,22 +1,99 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
+
+const movies = require("./data")
+
+function getAllDirectors() {
```

Comment:
One common comment on all the functions is that you are missing the argument of the function. This is why some of the test cases are failing. The test cases call your function and pass an array to it. However, the function is not accepting any inputs. Try instead:
`function getAllDirectors(movies) {`